docs/api/protocol.md

```diff
@@ -45,7 +45,7 @@ app.whenReady().then(() => {
 
   ses.protocol.registerFileProtocol('atom', (request, callback) => {
     const url = request.url.substr(7)
-    callback({ path: path.normalize(`${__dirname}/${url}`) })
+    callback({ path: path.normalize(path.join(__dirname, url)) })
```

Comment:
This is also just better path joining practice than the original example, glad the linter caught this one 🙂